server/application/application_test.go

```diff
@@ -1965,6 +1965,60 @@ func TestLogsGetSelectedPod(t *testing.T) {
 	})
 }
 
+func TestMaxPodLogsRender(t *testing.T) {
```

Comment:
I think this test probably needs to cover additional cases:

1) log query under max works
2) increasing max allows a request w/ higher number of pods to work
3) new max still prevents queries which break the limit

Reply:
@crenshaw-dev any other open points on this PR from your side?